src/rezplugins/shell/_utils/powershell_base.py

```diff
@@ -77,53 +78,32 @@ def get_syspaths(cls):
             cls.syspaths = config.standard_system_paths
             return cls.syspaths
 
-        # detect system paths using registry
-        def gen_expected_regex(parts):
-            whitespace = r"[\s]+"
-            return whitespace.join(parts)
-
         # TODO: Research if there is an easier way to pull system PATH from
         # registry in powershell
         paths = []
 
         cmd = [
-            "REG", "QUERY",
-            ("HKLM\\SYSTEM\\CurrentControlSet\\"
-             "Control\\Session Manager\\Environment"), "/v", "PATH"
+            "powershell",
+            '(Get-ItemProperty "HKLM:SYSTEM/CurrentControlSet/Control/Session Manager/Environment").Path',
         ]
 
-        expected = gen_expected_regex([
-            ("HKEY_LOCAL_MACHINE\\\\SYSTEM\\\\CurrentControlSet\\\\"
-             "Control\\\\Session Manager\\\\Environment"), "PATH",
-            "REG_(EXPAND_)?SZ", "(.*)"
-        ])
-
         p = Popen(cmd, stdout=PIPE, stderr=PIPE,
                   shell=True, text=True)
         out_, _ = p.communicate()
         out_ = out_.strip()
 
         if p.returncode == 0:
-            match = re.match(expected, out_)
-            if match:
-                paths.extend(match.group(2).split(os.pathsep))
+            paths.extend(out_.split(os.pathsep))
 
-        cmd = ["REG", "QUERY", "HKCU\\Environment", "/v", "PATH"]
-
-        expected = gen_expected_regex([
-            "HKEY_CURRENT_USER\\\\Environment", "PATH", "REG_(EXPAND_)?SZ",
-            "(.*)"
-        ])
+        cmd = ["powershell", "(Get-ItemProperty -Path HKCU:Environment).Path"]
```

Comment:
I think we could collapse this into a loop over the two different powershell calls and get rid of some code here

Reply:
@nerdvegas  Also a good point, i changed that, please check again if the new version is okay for you